aat/src/aat/resources/features/F-079/S-508.td.json

```diff
@@ -15,54 +15,29 @@
 			"Transfer-Encoding": "chunked"
 		},
 		"body": {
-			"id": "BEFTA_CASETYPE_2_1",
-			"description": "Demo Case BEFTA CASETYPE 2 1",
-			"version": {
-				"number": "[[ANYTHING_PRESENT]]",
-				"live_from": "2017-01-01T00:00:00.000+0000",
-				"live_until": null
-			},
-			"name": "Demo Case BEFTA CASETYPE 2 1",
-			"jurisdiction": {
-				"id": "BEFTA_JURISDICTION_2",
-				"name": "BEFTA JURISDICTION 2",
-				"description": "Content for the BEFTA Jurisdiction 2.",
-				"live_from": "2017-01-01T00:00:00.000+0000",
-				"live_until": null,
-				"case_types": []
-			},
-			"events": "[[ANYTHING_PRESENT]]",
-			"states": [
-				{
-					"id": "TODO",
-					"name": "To do",
-					"description": null,
-					"order": 1,
-					"title_display": null,
-					"acls": "[[ANYTHING_PRESENT]]"
-				},
-				{
-					"id": "IN_PROGRESS",
-					"name": "In progress",
-					"description": null,
-					"order": 2,
-					"title_display": null,
-					"acls": "[[ANYTHING_PRESENT]]"
-				},
-				{
-					"id": "DONE",
-					"name": "Done",
-					"description": null,
-					"order": 3,
-					"title_display": null,
-					"acls": "[[ANYTHING_PRESENT]]"
-				}
-			],
-			"acls": "[[ANYTHING_PRESENT]]",
-			"searchAliasFields": [],
-			"case_fields": "[[ANYTHING_PRESENT]]",
-			"printable_document_url": null,
-			"security_classification": "PUBLIC"
+      "id" : "FT_DateTimeFormats",
+      "description" : "CaseType for testing DateTime formats",
+      "version" : {
+        "number" : "[[ANY_INTEGER_NOT_NULLABLE]]",
+        "live_from" : "2017-01-01T00:00:00.000+0000",
+        "live_until" : null
+      },
+      "name" : "FT-DateTime Fomratting",
+      "jurisdiction" : {
+        "id" : "BEFTA_MASTER",
+        "name" : "BEFTA Master",
+        "description" : "Content for the BEFTA Master Jurisdiction.",
+        "live_from" : "2017-01-01T00:00:00.000+0000",
+        "live_until" : null,
+        "case_types" : [ ]
+      },
+      "events" : "[[ANYTHING_PRESENT]]",
+      "states" : "[[ANYTHING_PRESENT]]",
+      "acls" : "[[ANYTHING_PRESENT]]",
+      "searchAliasFields" : "[[ANYTHING_PRESENT]]",
+      "case_fields" : "[[ANYTHING_PRESENT]]",
+      "printable_document_url" : "[[ANYTHING_PRESENT]]",
```

Comment:
I know it may not feel like the most enjoyable thing to do, now that the vast majority of the work is nicely complete, but before we switch to something else and lose the chance to do it sooner, could we please verify these details, too?